jstp/src/test/java/com/metarhia/jstp/connection/JSTPConnectionTest.java

```diff
@@ -390,13 +394,15 @@ public void sendEscapedCharacters() throws Exception {
       @Override
       public void onConnected(boolean restored) {
         connection
-            .call("auth", "authorize", new JSArray(new Object[]{"+380962415331", "hellokitty1337"}),
+            .call("auth", "authorize",
+                new JSArray("+380962415331", "hellokitty1337"),
                 new ManualHandler() {
                   @Override
                   public void invoke(JSValue packet) {
                     JSObject userData = new JSObject();
                     userData.put("nickname", "\n\tnyaaaaaa'aaa'[((:’ –( :-)) :-| :~ =:O)],");
-                    connection.call("profile", "update", new JSArray(new Object[]{userData}),
+                    connection.call("profile", "update",
```

Comment:
To my mind, it looks worse after changes. Maybe, we should less strict about lines length? :)